src/cmap/auth/gssapi.ts

```diff
@@ -12,10 +12,23 @@ import {
 import { Callback, ns } from '../../utils';
 import { AuthContext, AuthProvider } from './auth_provider';
 
+/** @public */
+export const CanonicalizationProperties = Object.freeze({
```

Comment:
Would keeping these as `CanonicalizationValues` instead of "properties" (or perhaps `CanonicalizationSettings`) be better? Properties could imply keys that can be set on an object, but this could just be me

Reply:
Changes to `CanonicalizationValues`